modules/overlapblock/overlapblock_46_jepasco.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Innovation</title>
+    <style>
+        body {
+            background-image: url(images_and_icons/overlap_block_jepasco.jpeg);
```

Comment:
change `overlap_block_jepasco.jpeg` to `overlap_block_jepasco.jpg`

Reply:
Done!